app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
```

Comment:
According to the checklist, when importing multiple classes/modules, it is better to use parentheses. The current way of importing is not wrong, but it is recommended to follow the checklist to maintain consistency in the code style.